x-pack/plugins/enterprise_search/public/applications/shared/result/result_metadata.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SearchHit } from '@elastic/elasticsearch/lib/api/types';
+
+import { resultTitle } from './result_title';
+import { MetaDataProps } from './types';
+
+export const resultMetaData = (result: SearchHit, onDocumentDelete?: Function): MetaDataProps => ({
```

Comment:
question: Is it possible to merge this with result_title.ts? If you want to know how the metadata is really constructed you have to go back and forth between three files and it's not like the files are so big that you can't understand what's happening.

Reply:
Totally possible. I added this afterward writing `resultTitle` and didn't think about combining it.